### ISSUE TITLE
Ooh! Like a Smalltalk VM!

### ISSUE THREAD

**Issue by Trevoke** [baweaver/clairvoyant#2]
Shame this project looks abandoned, it could really help change the face of TDD in Ruby.

**Comment by baweaver** [comment]
More of the approach I was using wasn't really a great one, so I was waiting for inspiration on how to approach it differently.

I might try something along the lines of Pry rescue and inject into exceptions. Test fails, generate basic code and open $EDITOR with it there. Options to skip a particular test, move on, or otherwise.

Might tie into Guard for the loop / watch behavior and use reporters or something to catch errors using Rspecs output. 

Short version though, just hadn't come up with a good way to do it. The current way was unsustainable and works against Rspec instead of with it.
